bin/set_ngrok_url

```diff
@@ -0,0 +1,23 @@
+#!/usr/bin/env bash
+
+set -eo pipefail
+
+# Get current ngrok url from ngrok api launched with command "make ngrok"
+
+if docker-compose port ngrok 4040 &> /dev/null ; then
+    declare -r JQ="docker run --rm -i fundocker/jq:1.6"
+    NGROK_URL=$(
+        curl --no-progress-meter "$(docker-compose port ngrok 4040)/api/tunnels/command_line" | \
+        ${JQ} .public_url  | \
+        sed 's/"//g'
+    )
+    read -p "Apply to AWS config? (y/n) " -n 1 -r
+    echo
+    if [[ $REPLY =~ ^[Yy]$ ]]
+    then
+        sed -i "s+^TF_VAR_marsha_base_url=.*$+TF_VAR_marsha_base_url=$NGROK_URL+" ./src/aws/env.d/development
```

Comment:
Do we really need to modify this file or can we just pass the environment variable in the command line running terraform?

Reply:
There is at least 2 advantages to modify the file:
- you will not execute this script every time you want to deploy, only once. When you work on the AWS infra you apply terraform often
- You limit the modification to development file. Not for staging or production.

Reply:
but is it a problem if this script is included in bin/terraform and sets the environment variable only when in development and ngrok is started? It's probably a better way than modifying the file because it's always up-to-date no? Otherwise you will sure have cases where your ngrok was on but now it's off, and oh I have to modify the file manually and I don't understand what these guys have done :wink:

Reply:
What do you do when you are in production then ?

Reply:
you don't set it and it defaults to what's in the environment file?